tests/lib/tools/fs-state

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+show_help() {
+    echo "usage: fs-state wait-for-file [filename] [iterations] [sleep-time]"
+    echo "usage: fs-state mock-dir [direname]"
+    echo "usage: fs-state mock-dir-backup-real [direname]"
+    echo "usage: fs-state restore-dir [direname]"
```

Comment:
Typo, in all the file, `dirname`, not `direname`.

Reply:
done